ContextDesktop/src/test/java/com/github/migi_1/Context/main/TestMain.java

```diff
@@ -0,0 +1,193 @@
+package com.github.migi_1.Context.main;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import com.github.migi_1.Context.model.MainEnvironment;
+import com.github.migi_1.Context.screens.MainMenu;
+import com.github.migi_1.Context.utility.ProjectAssetManager;
+import com.jme3.asset.AssetManager;
+import com.jme3.input.InputManager;
+import com.jme3.network.serializing.Serializable;
+import com.jme3.renderer.RenderManager;
+import com.jme3.scene.Node;
+import com.jme3.system.AppSettings;
+
+/**
+ * TestSuite for the Main class.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Main.class, AppSettings.class, InputHandler.class,
+                ProjectAssetManager.class})
```

Comment:
Theres an easy way to PrepareForTest every class in our project instead of having to manually type this in. I can't remembed it by heart but you should check out some other test suites that do it.
This isn't something you need to change in this code, it's fine like this. Just a reminder for next time.